packages/zk-certificates/hardhat.config.ts

```diff
@@ -204,13 +225,20 @@ function getAccounts(): string[] {
   const accounts: string[] = [];
   // check if environment variables exist
   const warningMsg = ' env var not set, using random private key';
+
   if (process.env.GalaTestnetDeployerPrivateKey) {
+    console.log(
+      `adding 1 private key ${process.env.GalaTestnetDeployerPrivateKey}`,
```

Comment:
I think this should be removed to not print the keys on every console.

Reply:
removed.